js-miniapp-bridge/src/common-bridge.ts

```diff
@@ -99,6 +99,21 @@ export class MiniAppBridge {
       );
     });
   }
+
+  /**
+   * Associating requestCustomPermissions function to MiniAppBridge object
+   * @param {string[]} permissionTypes, Array of type of custom permissions that is requested
+   */
+  requestCustomPermissions(permissionTypes: string[]) {
```

Comment:
I guess we don't need to specify the datatype that is passed.

Reply:
In TypeScript, we need to specify the data type.
TS is defined as type object-oriented compile language which prevents the javascript any type style.

Reply:
Ah, I almost forgot that we changed to Typescript. My bad, sorry. @khairul-alam-licon please ignore.